### DISCUSSION TITLE
Proper logging alternative?

### DISCUSSION THREAD

**Discussion by muckelba** [zinclabs/zinc#59] [General]
Hey, i've just recently stumbled across this project and before i invest time setting this up, i want to ask first: Is zinc a real logserver alternative? Like can i setup filebeat/whatever and push my logs to it or is zinc designed for other stuff?

I was in the process of installing ELK but its quite complex and there's no free user management so i was searching for other options.

Apologies if that's not the right place to ask those questions.

**Comment by prabhatsharma** [comment]
You can certainly use zinc for logserver. I have not tested it with filebeat but zinc is known to work with fluentbit which is what I would recommend as a vendor neutral solution for shipping logs. Keep in mind that zinc is still in alpha stage and is under active development.
